confd/templates/docker-compose/docker-compose.tmpl

```diff
@@ -4,6 +4,21 @@
 version: '2.1'
 services:
 
+  grpc-stub:
```

Comment:
tbh this doesn't look like a mandatory part of the system, thus should be wrapped with `{{if not (exists "/no_grpc_stub")}}` thing

Reply:
fixed